OutOfSchool/IdentityServer/Controllers/AuthController.cs

```diff
@@ -14,12 +15,15 @@ public class AuthController : Controller
         private readonly SignInManager<User> _signInManager;
         private readonly UserManager<User> _userManager;
         private readonly IIdentityServerInteractionService _interactionService;
+        private readonly RoleManager<IdentityRole> _roleManager;
```

Comment:
we use a code convention for variables naming without underscore: _roleManager->roleManager